src/index.js

```diff
@@ -267,7 +267,7 @@ class Libp2p extends EventEmitter {
   }
 
   /**
-   * Dials to the provided peer. If successful, the known `PeerData` of the
+   * Dials to the provided peer. If successful, the known `Peer` data of the
    * peer will be added to the nodes `peerStore`
```

Comment:
Fix redundant wording
```suggestion
   * Dials to the provided peer. If successful, the known metadata of the
   * peer will be added to the nodes `peerStore`
```